samples/core/use_run_info/use_run_id.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import kfp
+import kfp.dsl as dsl
+
+
+def get_run_info(run_id: str):
+    '''Example of getting run info for current pipeline run'''
```

Comment:
nit: always use """triple double quotes""" around docstrings, and the summary line should be terminated by a period, question mark, or exclamation point.
https://google.github.io/styleguide/pyguide.html#381-docstrings

Reply:
Thanks, I'm seeing comments like this many times.
Can we automate this style check? (or even educate people how a formatter can automatically convert to our desired style)

Reply:
Yes, we should. I'm currently using an internal tool to do so. Haven't spend time trying the lint tool in google style guide yet. Feel free to open an issue and assign to me.